### DISCUSSION TITLE
Docker Install Finished and I just see a Mastodon Logo

### DISCUSSION THREAD

**Discussion by nedson** [mastodon/mastodon#21434] [General]
I just finished my mastodon install. I can't seem to find any errors when running 'docker logs mastodon_web_1'. The only thing interesting is this:

[ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: xx.xx.xx.xx

The blocked host is my servers external IP address.

When I login to my mastodon domain, I just see a mastodon logo (attached). Any pointers to what I might be doing wrong would be appreciated.

If it is relevant, I am running the latest version of Mastodon on Ubuntu 20.04 LTS.

![Screenshot from 2022-11-22 14-24-39](https://user-images.githubusercontent.com/1485243/203433849-03620ffd-30d5-4979-9fb9-4ec0bb995195.png)

**Comment by nedson** [comment]
Solved the problem:  had to fix the nginx.conf file:

I replaced the 404 with @proxy

Vim rocks:

:%s/=404/@proxy/g

Might want  to mention this somewhere - I haven't seen this mentioned in many guides. I don't know Docker very well, so perhaps this is obvious :-P

**Reply by vmstan** [reply]
Thank you, I was really confused by this and the instructions we not super clear to me. Working now!

**Comment by jdkruzr** [comment]
What is the equivalent fix for a non-Docker install, because I have the same problem installing from source. ETA: found this in /etc/nginx/sites-enabled/mastodon, tried changing it, had no effect.

**Comment by nedson** [comment]
You might make sure the correct permissions are set on your web root folder.
